core/trino-main/src/main/java/io/trino/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -1458,9 +1459,22 @@ private void printStatisticAggregationsInfo(
             }
 
             for (Map.Entry<ColumnStatisticMetadata, Symbol> columnStatistic : columnStatistics.entrySet()) {
+                String aggregationName;
```

Comment:
Is this functionality covered by any tests?

Reply:
No, there are no fine-grained tests for EXPLAIN.